sync/src/relayer/mod.rs

```diff
@@ -638,6 +670,9 @@ impl Relayer {
 
 impl CKBProtocolHandler for Relayer {
     fn init(&mut self, nc: Arc<dyn CKBProtocolContext + Sync>) {
+        if let NetFork::ForkInV1 = is_ckb2021(&nc, self.v2) {
```

Comment:
Would it be possible?
Why return directly?

Reply:
This is possible. When the client stops and restarts after completing the fork, the current state is the state after the fork. At this time, the v1 version of the relayer should be silent

Reply:
Gotcha.
Although I think it's better to put this check before initializing the protocol, [in launcher](https://github.com/nervosnetwork/ckb/pull/2765/files#diff-3d835c3d61b1399a2a6920bb7fa9d79484d35293d9c41eeb75ad388c49b58d3eR337-R341): if is_p2p_network_switch_enabled, do not init relay v1.